bin/block_generator/block_generator.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python3
+
+import argparse
+from contextlib import contextmanager
+import hashlib
+import json
+import jsonschema
+import logging
+from pathlib import Path
+import random
+import re
+import subprocess
+from time import sleep
+from typing import Dict
+
+# Interface for launching nil cluster RPC server
+class ClusterProcess:
```

Comment:
`RPCService`

Reply:
I think `RPC server` is also OK. Example: https://github.com/NilFoundation/nil/blob/main/rpc/transport/client.go#L20